esque/errors.py

```diff
@@ -154,136 +139,5 @@ class TopicConfigNotValidException(YamaleValidationException):
     3: TopicDoesNotExistException,
     36: TopicAlreadyExistsException,
     -191: EndOfPartitionReachedException,
+    38: InvalidReplicationFactorException,
 }
-
-
-class ErrorCode(Enum):
```

Comment:
Can we keep those in a comment? For future reference.

Reply:
how about I do not use the plain ints in `ERROR_LOOKUP` but the constants from `KafkaError`? Then people can see where it comes from and look there.